osbrain/tests/common.py

```diff
@@ -1,10 +1,16 @@
+import sys
+import time
 import pytest
 
 from osbrain import run_agent
 from osbrain import run_logger
 from osbrain import run_nameserver
 from osbrain.helper import sync_agent_logger
 
+skip_on_windows = pytest.mark.skipif(sys.platform == 'win32',
+                                     reason='Not supported on windows')
+skip_windows_addr = pytest.mark.skipif(sys.platform == 'win32',
+                                       reason='Windows allows port reuse')
```

Comment:
Just for consistency I'd name both `skip_on_windows...` or `skip_windows...` (it is easier to `grep`).